src/CosmosEditorManager.ts

```diff
@@ -76,7 +76,11 @@ export class CosmosEditorManager {
     }
 
     private async updateEditor(data: {}, textEditor: vscode.TextEditor): Promise<void> {
-        await util.writeToEditor(textEditor, JSON.stringify(data, null, 2));
+        if (typeof data === "string") {
+            await util.writeToEditor(textEditor, <string>data);
+        } else {
+            await util.writeToEditor(textEditor, JSON.stringify(data, null, 2));
+        }
```

Comment:
1. The Procedure and ProcedureMeta types contain a string for the the StoredProcedure body.  Passing it through JSON stringify would parse 
```
function() {
// Comment
}
```
as `function() {\n// Comment\n}\n`

2. I didn't change the function signature because I wanted to keep this change small. 

Can instead write as `typeof data === "string" ? <string>data :  JSON.stringify(data, null, 2)`. While more succinct, it still feels spaghetti-ish.